ui/src/i18n/pl.json

```diff
@@ -225,10 +225,10 @@
   "plugins.manage.message_uninstall_remove_confirmation": "Na pewno odinstalować {{ pluginName }}?",
   "plugins.manage.message_uninstall_remove_plugin_config": "Usunąć również konfigurację tej wtyczki?",
   "plugins.manage.modal_unverified_message": "Niezweryfikowane wtyczki Homebridge wskazują, że nie zostały poddane procesowi przeglądu przez zespół projektowy Homebridge. Brak weryfikacji niekoniecznie oznacza nieskuteczność lub obawy dotyczące bezpieczeństwa. Autorzy mogli zdecydować się nie szukać weryfikacji z różnych powodów.",
-  "plugins.manage.modal_unverified_subtitle": "{{ pluginName }} jest niezweryfikowane.",
+  "plugins.manage.modal_unverified_subtitle": "{{ pluginName }} jest niezweryfikowany.",
```

Comment:
"niezweryfikowany" or "niezweryfikowana"

Reply:
Camera UI jest niezweryfikowana, 
WebOS Control jest niezweryfikowany, 
Wtyczki są niezweryfikowane

I think first two are correct

Reply:
<img width="722" alt="Zrzut ekranu 2024-07-25 o 09 22 55" src="https://github.com/user-attachments/assets/03895862-4c98-4f51-89d1-b1b5443ce72b">

This is not displayed next to the plugin name. So for me "niezweryfikowana" or "nie zweryfikowano" (with space, cause it is verb).